flocker/acceptance/testtools.py

```diff
@@ -771,15 +781,20 @@ def get_test_cluster(node_count=0):
                        "{existing} node(s) are set.".format(
                            necessary=node_count, existing=len(agent_nodes)))
 
+    certificates_path = FilePath(
+        environ["FLOCKER_ACCEPTANCE_API_CERTIFICATES_PATH"])
```

Comment:
It would be nice to have this kind of environment lookup code in just one place (same applies to the existing code here, eg the code looking up `FLOCKER_ACCEPTANCE_CONTROL_NODE` and duplicating the missing-case handling that's also present in `get_nodes`).

Refactor or file an issue and add a link.